### DISCUSSION TITLE
Using cache while training

### DISCUSSION THREAD

**Discussion by Gunj23** [ultralytics/ultralytics#2110] [General]

**Comment by glenn-jocher** [comment]
@Gunj23 hello!

When training YOLOv8 models, caching is automatically implemented to speed up subsequent epochs after the first one. During the first epoch, the dataset is loaded into memory, which may take some time depending on the size of your dataset. However, once this initial loading is complete, the cached data is used for faster access in the following epochs, reducing the overall training time.

If you're experiencing issues with caching or have specific questions about its behavior, please provide more details, and I'll be happy to assist you further. Happy training! 😊🚀
